Multisig/UI/App/UIKit/UI Library/ReusableViews/AddressInfoView.swift

```diff
@@ -8,18 +8,29 @@
 
 import UIKit
 
-class AddressWithTitleView: UINibView {
+struct AddressInfo {
+    let address: Address
+    let label: String?
+}
+
+class AddressInfoView: UINibView {
     @IBOutlet private weak var identiconView: UIImageView!
     @IBOutlet private weak var textLabel: UILabel!
     @IBOutlet private weak var detailLabel: UILabel!
-    @IBOutlet private weak var button: UIButton!
+    @IBOutlet private weak var copyButton: UIButton!
+    @IBOutlet private weak var disclosureButton: UIButton!
```

Comment:
How come do we have disclosure button in address info view? What is it for?

Reply:
This is for view address details or delete button (imported key address)

Reply:
I think the name is misleading - can we name it "detailButton"?